### PR TITLE
Add two helper functions to help with changing attributes on grouped objects

### PR DESCRIPTION
**Pull request by VictorHagsand**
Thought this might be useful since its currently a bit tricky to make changes to a single or all selected objects and get the recalculated correctly. Its possible to add and remove objects and get everything recalculated, so this is a compliment to that.

### CODE REVIEW

File: src/shapes/group.class.js [hunk 1]
```diff
@@ -188,6 +188,49 @@
...
+    
+    /**
+     * Sets a property to all objects in a group; Then recalculates group's dimension, position.
+     * @param {String} prop Property to set
+     * @param {String} value Value to set
+     * @return {fabric.Group} thisArg
+     * @chainable
+     */
+    setObjectsWithUpdate: function(prop, value) {
+        this._moveFlippedObject(object);
```
Comment (kangax) [inline src/shapes/group.class.js]: Which `object`?

**Comment by asturur** [comment]
I have some doubts on this.
Other than _moveFlippedObject is deprecated, i do not see why someone should recalculate the object when changing "fill" for example.
We have .realizeTransform that is chainable and someone could just call group.objects[x].setColor().realizeTransform() for example.

**Comment by kangax** [comment]
I think this kind of API is useful but the implementation could use some tweaking. @asturur, like you say, we should use `realizeTransform` and possibly _only_ when changing certain set of properties (for perf. reasons)

**Comment by VictorHagsand** [comment]
Can you point me at realizeTransform() in the docs or is that a undocumented function as many others? The docs are in a pretty bad state sadly and its really hard to find stuff : /

**Comment by asturur** [comment]
Yes docs are in bad state.

Please check 
https://github.com/asturur/fabric.js/blob/fix-group-transform/src/shapes/group.class.js#L327

moveFlippedObject is phasing out, realizeTransform is now the old "resetObjectPosition".
But we had it before as the combination of _moveFlippedObject and resetObjectPosition.

**Comment by VictorHagsand** [comment]
Thank you : )

**Comment by kangax** [comment]
Andrea, could you please add this to docs.

**Comment by asturur** [comment]
When pr is merged and is stable, i will check even docs. Now is not the moment. We are changing lot of stuff.

**Pull request closed without merging**